Admin Logs not HTML encoded

## Description of bug
When viewing the admin logs, the exception content is not HTML encoded.  For example, if an exception's stack trace includes a generic type or method (e.g. `List<string>`), then the generic type inside the brackets is interpreted as an HTML element instead of text.

## Steps to reproduce
List the steps to reproduce the behavior:
1. Install 
[Engage.GenericsError_1.0.0_Install.zip](https://github.com/dnnsoftware/Dnn.Platform/files/3570013/Engage.GenericsError_1.0.0_Install.zip)
2. Add module to page
3. View _Admin Logs_

## Current result
A _Module Load Exception_ is present in the logs with the message
>error CS1061: '`List`' does not contain a definition for '`DoSomethingFun`' and no accessible extension method '`DoSomethingFun`' accepting a first argument of type '`List`' could be found (are you missing a using directive or an assembly reference?)

## Expected result
A _Module Load Exception_ is present in the logs with the message
>error CS1061: '`List<char>`' does not contain a definition for '`DoSomethingFun`' and no accessible extension method '`DoSomethingFun`' accepting a first argument of type '`List<char>`' could be found (are you missing a using directive or an assembly reference?)

## Screenshots
![Log message with missing generic types](https://user-images.githubusercontent.com/59507/64182195-6d0e1380-ce2d-11e9-8be0-9214997b0541.png)

## Affected version
<!-- 
Please add X in at least one of the boxes as appropriate. In order for an issue to be accepted, a developer needs to be able to reproduce the issue on a currently supported version. If you are looking for a workaround for an issue with an older version, please visit the forums at https://dnncommunity.org/forums
-->

* [ ] 10.0.0 nightly build
* [ ] 9.4.1 nightly build
* [x] 9.4.0 release candidate
* [x] 9.3.2 latest supported release

--- a/DNN Platform/Library/Services/Exceptions/ExceptionInfo.cs
+++ b/DNN Platform/Library/Services/Exceptions/ExceptionInfo.cs
@@ -22,8 +22,8 @@
 
 using System;
 using System.IO;
+using System.Net;
 using System.Text;
-using System.Web.UI.WebControls;
 using System.Xml;
 
 #endregion
@@ -180,23 +180,23 @@ namespace DotNetNuke.Services.Exceptions
 		public override string ToString()
 		{
 			var str = new StringBuilder();
-			str.Append("<p><strong>AssemblyVersion:</strong>" + AssemblyVersion + "</p>");
+			str.Append("<p><strong>AssemblyVersion:</strong>" + WebUtility.HtmlEncode(AssemblyVersion) + "</p>");
 			str.Append("<p><strong>PortalId:</strong>" + PortalId + "</p>");
 			str.Append("<p><strong>UserId:</strong>" + UserId + "</p>");
 			str.Append("<p><strong>TabId:</strong>" + TabId + "</p>");
-			str.Append("<p><strong>RawUrl:</strong>" + RawUrl + "</p>");
-			str.Append("<p><strong>Referrer:</strong>" + Referrer + "</p>");
-			str.Append("<p><strong>UserAgent:</strong>" + UserAgent + "</p>");
-			str.Append("<p><strong>ExceptionHash:</strong>" + ExceptionHash + "</p>");
-			str.Append("<p><strong>Message:</strong>" + Message + "</p>");
-			str.Append("<p><strong>StackTrace:</strong><pre>" + StackTrace.Replace(") at ", ")<br/>at ") + "</pre></p>");
-			str.Append("<p><strong>InnerMessage:</strong>" + InnerMessage + "</p>");
-			str.Append("<p><strong>InnerStackTrace:</strong><pre>" + InnerStackTrace.Replace(") at ",")<br/>at ") + "</pre></p>");
-			str.Append("<p><strong>Source:</strong>" + Source + "</p>");
-			str.Append("<p><strong>FileName:</strong>" + FileName + "</p>");
+			str.Append("<p><strong>RawUrl:</strong>" + WebUtility.HtmlEncode(RawUrl) + "</p>");
+			str.Append("<p><strong>Referrer:</strong>" + WebUtility.HtmlEncode(Referrer) + "</p>");
+			str.Append("<p><strong>UserAgent:</strong>" + WebUtility.HtmlEncode(UserAgent) + "</p>");
+			str.Append("<p><strong>ExceptionHash:</strong>" + WebUtility.HtmlEncode(ExceptionHash) + "</p>");
+			str.Append("<p><strong>Message:</strong>" + WebUtility.HtmlEncode(Message) + "</p>");
+			str.Append("<p><strong>StackTrace:</strong><pre>" + WebUtility.HtmlEncode(StackTrace)?.Replace(") at ", ")<br/>at ") + "</pre></p>");
+			str.Append("<p><strong>InnerMessage:</strong>" + WebUtility.HtmlEncode(InnerMessage) + "</p>");
+			str.Append("<p><strong>InnerStackTrace:</strong><pre>" + WebUtility.HtmlEncode(InnerStackTrace)?.Replace(") at ",")<br/>at ") + "</pre></p>");
+			str.Append("<p><strong>Source:</strong>" + WebUtility.HtmlEncode(Source) + "</p>");
+			str.Append("<p><strong>FileName:</strong>" + WebUtility.HtmlEncode(FileName) + "</p>");
 			str.Append("<p><strong>FileLineNumber:</strong>" + FileLineNumber + "</p>");
 			str.Append("<p><strong>FileColumnNumber:</strong>" + FileColumnNumber + "</p>");
-			str.Append("<p><strong>Method:</strong>" + Method + "</p>");
+			str.Append("<p><strong>Method:</strong>" + WebUtility.HtmlEncode(Method) + "</p>");
 			return str.ToString();
 		}
 

